scripts/host_modules/ztp_handler.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+'''
+Copyright 2019 Broadcom. The term "Broadcom" refers to Broadcom Inc.
```

Comment:
Copyright assigned to Broadcom? Shouldn't we own the copyright for our code?

Reply:
A part of code code in ztp_handler.py file is taken from broadcom's implementation of ztp(in the host), so their copyright is left as it is.